src/js/components/ServicesBreadcrumb.js

```diff
@@ -85,6 +92,8 @@ class ServicesBreadcrumb extends React.Component {
 }
 
 ServicesBreadcrumb.propTypes = {
+  className: React.PropTypes.string,
+  h4ClassName: React.PropTypes.string,
```

Comment:
instead of calling this `h4ClassName` can we call this headerClassName or something? so it's not so tied to h4